jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/util/HealthCheck.java

```diff
@@ -78,4 +84,27 @@ public Response ping() {
         return Response.ok("OK").build();
     }
 
+
+    /**
+     * Endpoint to verify scm and version details
+     *
+     * @return {@link Attributes}
+     * @statuscode 200 Successful response for your request with manifest's attributes
+     * @statuscode 404 Not found version information
+     */
+    @GET
+    @Path("/version")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response manifestDetails(@Context HttpServletRequest request) {
+
+        final ServletContext context = request.getSession().getServletContext();
+
+        try {
+            final InputStream manifestStream = context.getResourceAsStream("/META-INF/MANIFEST.MF");
```

Comment:
As a rule of thumb, it would be better to always close streams.
I would suggest using 
`try(InputStream manifestStream = context.getResourceAsStream("/META-INF/MANIFEST.MF"))`